src/core/accountAddress.ts

```diff
@@ -21,6 +21,8 @@ export enum AddressInvalidReason {
   INVALID_PADDING_ZEROES = "INVALID_PADDING_ZEROES",
 }
 
+export type AccountAddressInput = HexInput | AccountAddress;
```

Comment:
can we have it in the global type file? src/types/index.ts

Reply:
Sure, will do

Reply:
This is... impossible, and creates a dependency loop that's really I think impossible to get rid of since it then depends on itself

Reply:
do we use HexInput anywhere now? if not, can remove this type and just use this as
```
export type AccountAddressInput = string | Uint8Array | AccountAddress;
```

Reply:
We use HexInput for other things like hashes, and bytes